truss/templates/copy_cache_files.Dockerfile.jinja

```diff
@@ -1,5 +1,5 @@
 {% for file in cached_files %}
-    {%- if credentials_exists %}
+    {%- if file.startswith("/app/model_cache/") %}
```

Comment:
I think this would be a little bit cleaner if we could have these templates be more logicless. Is there something else that we can check here? It's not clear from reading this template file what the implications of the file being named /app/model_cache/ are

Reply:
This is the cache copying mechanism. The HuggingFace files have a special root directory while other files do not. Let me think about this and get back.

Reply:
It might make sense to keep the `.startswith` but instead just use `/app/` instead. If the file is relative to `app` we want to copy it to the same place.

Reply:
> The HuggingFace files have a special root directory while other files do not. Let me think about this and get back."

This assumption is baked in here & implicit but not made explicit anywhere. In the future someone might wonder why we're doing this. Here's an example of an approach that makes this _explicit_ instead of _implicit_:

```
@dataclass
class CachedFile:
    source: str
    dst: str

cached_files = [
  # Huggingface files have a special root directory, while the others do not.
  # Being in app/model_cache implies that it is not a huggingface file
  CachedFile(src=... if file.startswith(...) , dst=...) for file in files
]

data = {
   ...
  cached_files: cached_files
  ...
}

render_template(data)
```